tale_studio/utils.py

```
@@ -0,0 +1,132 @@
+):
+    response = None
+    while True:
+        try:
+            response = novel_completion(
+                prompt=prompt,
+                model_settings=model_settings,
+                system_prompt=system_prompt,
+            )
+            output = parse_json_output(response)
+            break
+        except Exception:
```

Ruff (0.14.4)

Do not catch blind exception: `Exception`

(BLE001)